changelogs/fragments/5022-lastpass-lookup-cleanup.yml

```
@@ -0,0 +1,2 @@
+trivial:
+ - lastpass - use config manager API in tests; improve documentation.
```

Test and documentation changes do not need a changelog fragment. But the main change of this PR is something that must be mentioned:
```suggestion
minor_changes:
 - lastpass - use config manager for handling plugin options (https://github.com/ansible-collections/community.general/pull/5022).
```